pkgs/development/libraries/fortify-headers/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation {
+  pname = "fortify-headers";
+  version = "1.1alpine1";
+
+  # upstream only accessible via git - unusable during bootstrap, hence
+  # extract from the alpine package
+  src = fetchurl {
+    url = "https://dl-cdn.alpinelinux.org/alpine/v3.17/main/x86_64/fortify-headers-1.1-r1.apk";
```

Comment:
Shouldn't version number appear in the URL?

Reply:
I'm the one calling this `1.1alpine1` - the `r1` is not an official designation, just alpine's release of the package. And it _does_ differ from upstream's package slightly, including a patch to fix `ppoll` on some systems https://git.alpinelinux.org/aports/commit/main/fortify-headers?id=4f60e618352e581f7f77a3842e29141da8992d5f. `r3` in fact includes patches for clang support, but I'm not using that one yet because I can't find a stable url for it, only appearing in the edge release.